src/QueryBuilder.php

```diff
@@ -34,6 +34,7 @@ class QueryBuilder extends ConfigurableService
 
     private const STANDARD_FIELDS = [
         'class',
+        'parents',
```

Comment:
Can you please put a @TODO to avoid us to forward elastic search syntax from FE directly?

Reply:
here? I think we should add this to frontend